src/features/wallet/TransactionHistoryModal.tsx

```diff
@@ -0,0 +1,264 @@
+import crossIcon from '../../common/assets/icons/ico-cross.svg'
+import addressbookIcon from '../../common/assets/icons/ico-addressbook.svg'
+import user2Icon from '../../common/assets/icons/ico-user2.svg'
+
+export type TransactionHistoryModalProps = {
+  isOpen: boolean
+  handleClose: () => void
+}
+
+const TransactionHistoryModal: React.FC<TransactionHistoryModalProps> = ({
+  isOpen,
+  handleClose,
+}) => {
+  const [selectedOption, setSelectedOption] = useState<
+    'all' | 'received' | 'sent' | 'in progress'
+  >('all')
+  const [dates, setDates] = useState<momentRange.DateRange>()
+  const [address, setAddress] = useState<TOption | null>({
+    label: 'All',
+    value: '',
+  })
+  const [recipient, setRecipient] = useState<TOption | null>({
+    label: 'All',
+    value: '',
+  })
+
+  const onSelectDateRange = (dates: OnSelectCallbackParam) => {
+    setDates(dates as momentRange.DateRange)
+  }
+  return (
+    <TitleModal
+      isOpen={isOpen}
+      handleClose={() => handleClose()}
+      size='7xl'
+      title='Transaction history'
+    >
+      <div>
+        <div className='flex text-gray-71 text-sm'>
+          <div className='w-2/3 flex'>
+            <div className='w-1/3 pr-10'>
+              <div>Time range</div>
+              <DateRangeSelector value={dates} onSelect={onSelectDateRange} />
+            </div>
+            <div className='w-1/3 pr-6'>
+              <div className='mb-1'>Source address</div>
+              <Select
+                label={<img src={addressbookIcon} className='ml-4 mr-2' />}
+                className='text-gray-2d w-112px'
+                selected={address}
+                options={sourceAddresses}
+                onChange={setAddress}
+              />
+            </div>
+            <div className='w-1/3 pr-6'>
+              <div className='mb-1'>Recipients</div>
+              <Select
+                label={<img src={user2Icon} className='ml-4 mr-2' />}
+                className='text-gray-2d w-112px'
+                selected={recipient}
+                options={recipients}
+                onChange={setRecipient}
+              />
+            </div>
+          </div>
+          <div className='w-1/3 flex justify-end items-center space-x-4 pt-6'>
+            <Radio
+              isChecked={selectedOption === 'all'}
+              clickHandler={() => setSelectedOption('all')}
+            >
+              All
+            </Radio>
+            <Radio
+              isChecked={selectedOption === 'received'}
+              clickHandler={() => setSelectedOption('received')}
+            >
+              Received
+            </Radio>
+            <Radio
+              isChecked={selectedOption === 'sent'}
+              clickHandler={() => setSelectedOption('sent')}
+            >
+              Sent
+            </Radio>
+          </div>
+        </div>
+        <div className='h-409px overflow-y-auto scrollbar mt-3 pr-2'>
```

Comment:
When removing a global style, I mean scrollbar, please make sure to remove its usages.

Reply:
Sure, I'm not using scrollbar anymore. and this is my work as well. I will remove it.